doc/articles/getting-started-tutorial-1.md

```diff
@@ -0,0 +1,412 @@
+---
+title: Tutorial - Creating a Single Page App with Uno
+description: How to get started developing Cross Platform apps with Uno
+---
+
+# Create a Single Page App with Uno
+
+[Download Sample](https://github.com/nventive/Uno.GettingStarted)
+
+In this tutorial you will learn how to:
+
+- Add the Uno Project Templates to Visual Studio
+- Create a new Project with Uno
+- Learn basics on Model Binding
+
+The tutorial walks through creating a cross platform application with Uno, which enables you to see a single Issue entry.
+
+### Prerequisites
+
+- Visual Studio 2019 (latest release), with the **Mobile development with .NET** workload installed.
+- Knowledge of C#.
+- (optional) A paired Mac to build the iOS project.
+
+For more information about these prerequisites, see [Installing Xamarin](https://docs.microsoft.com/en-us/xamarin/get-started/installation/index.md). For information about connecting Visual Studio 2019 to a Mac build host, see [Pair to Mac for Xamarin.iOS development](https://docs.microsoft.com/en-us/xamarin/ios/get-started/installation/windows/connecting-to-mac/index.md).
+
+##### On Windows
+
+- Running WASM Project using Python
```

Comment:
is it still worth mentioning running using Python ? Our solution template doesn't offer it anymore.

Reply:
Good point, I'll update this part.